### PR TITLE
Update deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md

### PR DESCRIPTION
**Pull request by lucabandMSFT**
## Category

- [x] Content fix
- [ ] New article
- [x] Example checked item (*delete this line*)

## What's in this Pull Request?

Added additional information in to speed up the 24 hour delay period for new developed add-ins.

### CODE REVIEW

File: docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md
Comment (jansenbe) [inline docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md:69]: @lucabandMSFT : think we should be include the sample script here

Comment (jansenbe) [reply docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md:69]: Sample script to include can be this:

```PowerShell
Connect-MgGraph -Scopes Application.ReadWrite.All

$sharepointPrincipal = Get-MgServiceprincipal -Filter "AppId eq '00000003-0000-0ff1-ce00-000000000000'"

$sharepointPrincipal | fl

$replyUrls = $sharepointPrincipal.ReplyUrls

$replyUrls += "https://contoso-79d42f062409ae.sharepoint.com/_forms/default.aspx"

Update-MgServiceprincipal -ServicePrincipalId $sharepointPrincipal.Id -ReplyUrls $replyUrls
```

Comment (andrewconnell) [reply docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md]: I fixed a few things in the doc, but I'll let you guys decide on the script that @jansenbe referenced so I don't inject myself into the process...

Comment (lucabandMSFT) [reply docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md]: @andrewconnell, @jansenbe : question for both of you, if I include the script, is it ok to use a fake tenant name like contoso.sharepoint.com? Also, if I do that, should I include a full example of an error with the fake (contoso) name as the error will have both the name of the tenant and the appID?

Comment (andrewconnell) [reply docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md]: @lucabandMSFT said:
> if I include the script, is it ok to use a fake tenant name like contoso.sharepoint.com

Yup, in fact that's the exact one I'd use

> if I do that, should I include a full example of an error with the fake (contoso) name as the error will have both the name of the tenant and the appID?

If you think it would be helpful, sure... but it can also be noise in the doc if it's a long error snippet. My approach is to only add it if it adds value, but to err on not adding noise. If the doc gets questions or warrants more detail, then sure. Then again, it can't hurt to add it... but maybe add it as an Appendix at the end of the doc?

Comment (lucabandMSFT) [reply docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md]: Thanks @andrewconnell. Let me rewrite it in a bit.

**Review by andrewconnell — APPROVED** [review]

File: docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md
Comment (andrewconnell) [reply docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md]: edits... waiting for the automated build to complete to verify formatting looks correct... will merge if all is good

**Pull request merged**
